src/autowiring/CoreThreadMac.cpp

```diff
@@ -11,8 +11,17 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <iostream>
+#include <pthread.h>
 #include THREAD_HEADER
 
+// Missing definitions from pthreads.h
+#if !defined(PTHREAD_MIN_PRIORITY)
+#define PTHREAD_MIN_PRIORITY  0
+#endif
+#if !defined(PTHREAD_MAX_PRIORITY)
+#define PTHREAD_MAX_PRIORITY 31
```

Comment:
This was done 20 months ago and never committed. It was an initial attempt at the problem. Those numbers probably came from Linux documentation. It is unfortunate that it isn't defined. If 63 is correct, then that should be used instead.